### ISSUE TITLE
Bugs in LAW=7 and LAW=9

### ISSUE THREAD

**Issue by HunterBelanger** [NuclearData/ACEFormat#19]
For LAW=7 (Maxwellian), the definition of the normalization constant I does not match the definition found in the ENDF manual.

For LAW=9 (Evaporation), the form of the distribution does not agree with that found in the ENDF manual (should be E_{out} in the numerator, and not \sqrt{E_{out}}).  There is a note however which explicitly states that the form of the distribution for Maxwellian and Evaporation should be the same. Assuming that the ENDF version of these distributions is correct, then this note is also inaccurate.

I have fixes prepared for both of these, I just want to make sure that the LAW=9 discrepancy is indeed a bug (as the explicit note makes me somewhat suspicious).

**Comment by paulromano** [comment]
@HunterBelanger I agree with you on both counts. The form of the distribution for LAW=9 is definitely not the same as LAW=7, so I'm also puzzled as to why there's a note saying they should be the same.

**Comment by HunterBelanger** [comment]
@paulromano Thanks for confirming my suspicions ! I will go ahead and submit my fix for this then.
